app/models/AuthorityResource.java

```diff
@@ -187,6 +187,33 @@ public String gndRelationEdges() {
 		return Json.toJson(result).toString();
 	}
 
+	public Integer getBirthYear() {
+		String[] parts = fieldValues("dateOfBirth-dateOfDeath", json)
+			.collect(Collectors.joining())
+			.split("–");
+		return Integer.parseInt(parts[0]);
```

Comment:
Instead of first getting the composed form `dateOfBirth-dateOfDeath` and splitting that, we can get the `dateOfBirth` directly using `fieldValues("dateOfBirth", json)`. We also need to make sure to handle missing birth dates here (see e.g. https://test.lobid.org/gnd/1359893474). Finally, like you did in `getDeathYear()`, we should get only the year, but we can use the existing `year()` function for that. So e.g.:

```suggestion
		return fieldValues("dateOfBirth", json).findFirst()
				.map(AuthorityResource::year)
				.map(Integer::parseInt)
				.orElse(null);
```